src/Release/MysqlReleaseToggleCollectionGateway.php

```diff
@@ -36,11 +37,7 @@ public function __construct(Connection $connection )
      */
     public function getTogglesForRelease( $releaseId )
     {
-        $toggles = [ ];
         $data = $this->connection->fetchAll( 'SELECT * FROM `toggle` WHERE release_id = ?', [ $releaseId ] );
-        foreach ( $data as $row ) {
-            $toggles[] = new Toggle( $row[ 'name' ], $releaseId, (bool) $row[ 'is_active' ] );
-        }
-        return $toggles;
+        return $this->getAllTogglesForGivenSqlStatement( $data );
```

Comment:
Hmm... I feel as if the name "get all toggles for given sql statement" isn't quite right, as it doesn't take an sql statement, it takes a set of results. 

``` getAllTogglesForGivenRows ``` maybe? Not sure. But you're not passing in an sql statement